src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporterOptions.cs

```diff
@@ -90,4 +90,11 @@ public int MetricExportIntervalMilliseconds
             this._prepopulatedMetricDimensions = copy;
         }
     }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether or not OpenTelemetry SDK metric
+    /// name validation should be disabled. Default value: <see
+    /// langword="false"/>.
+    /// </summary>
+    public bool DisableMetricNameValidation { get; set; }
```

Comment:
any way to avoid this? Like some env_variable from which we replace regex, instead of exposing a public property?

Reply:
or something part of connection string, and undocumented.

Reply:
@cijothomas I'm fine with either of those @dpk83 you have a preference?

Reply:
I am fine with any mechanism here because ultimately it's anyways using reflection and OTel SDK isn't exposing this ability publicly in the first place.

Reply:
I vote for any solution that can give a clear contract, rather than having R9 (or other consumers) to do hacks/reflections.

Reply:
@reyang Define clear contract 😄 Good with EnvVar or connection string switch for that? Or are you saying you prefer the existing bool switch?

Reply:
EnvVar, ConnectionString, this PR - all are clear contracts!
I just prefer to not make it part of the main options and clutter it, as this is really a hack, so making it in options seems less attractive to me.

Reply:
Updated to use a connection string flag

Reply:
> I just prefer to not make it part of the main options and clutter it, as this is really a hack, so making it in options seems less attractive to me.

💯